sympy/sets/sets.py

```diff
@@ -729,9 +729,10 @@ def __new__(cls, start, end, left_open=False, right_open=False):
             raise ValueError("Non-real intervals are not supported")
 
         # evaluate if possible
+        empty_interval = (end - start).is_negative
```

Comment:
Instead of `empty_interval`, `is_empty` might be a more suitable name to use here.